pandas/core/groupby/groupby.py

```diff
@@ -869,6 +869,9 @@ def _cython_transform(self, how, numeric_only=True, **kwargs):
 
         return self._wrap_transformed_output(output, names)
 
+    def _wrap_aggregated_output(self, output, names=None):
```

Comment:
This func was also not a direct copy paste because it existed in both classes. What was in NDFrameGroupBy was moved to GroupBy (can ref last commit)